excellent/evaluator_test.go

```diff
@@ -221,7 +223,7 @@ func TestEvaluateTemplate(t *testing.T) {
 		"dec1":    types.RequireXNumberFromString("1.5"),
 		"dec2":    types.RequireXNumberFromString("2.5"),
 		"words":   types.NewXText("one two three"),
-		"array":   types.NewXArray(types.NewXText("one"), types.NewXText("two"), types.NewXText("three")),
+		"array1":  types.NewXArray(types.NewXText("one"), types.NewXText("two"), types.NewXText("three")),
```

Comment:
now you can't have (top-level) names in the context that conflict with function names